projects/optic/src/__tests__/integration/api-add.test.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  test,
+  expect,
+  describe,
+  jest,
+  beforeEach,
+  afterEach,
+} from '@jest/globals';
+import {
+  runOptic,
+  setupWorkspace,
+  normalizeWorkspace,
+  setupTestServer,
+  run,
+} from './integration';
+jest.setTimeout(30000);
+
+function sanitizeOutput(out: string) {
+  return out.replace(/[a-zA-Z0-9]{8}:/g, 'COMMIT-HASH:');
+}
+
+setupTestServer(({ url, method }) => {
+  return JSON.stringify({});
+});
+
+describe('optic api add', () => {
+  let oldEnv: any;
+  beforeEach(() => {
+    oldEnv = { ...process.env };
+    process.env.LOG_LEVEL = 'info';
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+    process.env = { ...oldEnv };
+  });
+
+  describe('no interaction', () => {
```

Comment:
I'm not really sure how we can get the integration test to run with interaction / what needs to be changed - I just forced this test to not prompt for anything